core/include/arp_poisoning/public.h

```diff
@@ -14,8 +15,8 @@ void allOutPoison(const AllOutPoisoningOptions &options);
 struct SilentPoisoningOptions {
     std::string ifaceIpOrName;
     std::optional<std::string> attackerMac;
-    std::optional<std::string> victimIp;
-    std::string ipToSpoof;
+    std::unordered_set<std::string> victimIp;
```

Comment:
Make this plural too
```suggestion
    std::unordered_set<std::string> victimIps;
```